packages/react-components/react-charts-preview/library/src/components/LineChart/LineChart.test.tsx

```diff
@@ -258,13 +241,6 @@ describe('Render calling with respective to props', () => {
 describe('LineChart - mouse events', () => {
   let root: HTMLDivElement | null;
 
-  beforeEach(() => {
-    sharedBeforeEach();
-
-    root = document.createElement('div');
```

Comment:
why removing this code

Reply:
I push the new updates; I revert back the previous code and now only removed the "sharedBeforeEach();" line.